src/Seabug/Contract/CnftMint.purs

```diff
@@ -69,13 +69,16 @@ mintCnft (MintCnftParams params) = do
   unbalancedTx <- liftedE $ Lookups.mkUnbalancedTx lookups constraints
   policyHash <- liftedM "Could not get minting policy hash" $ liftAff $
     mintingPolicyHash policy
+  name <- liftContractM "Invalid CIP25 NFT name. The name is probably too long."
+    $ mkCip25String params.name
   unbalancedTxWithMetadata <- setTxMetadata unbalancedTx $ Cip25Metadata
     [ Cip25MetadataEntry
         { policyId: policyHash
-        , assetName: tn
-        , imageUris: NonEmpty.singleton params.imageUri
+        , assetName: wrap tn
+        , name
+        , image: params.imageUri
         , mediaType: Nothing
-        , description: [ params.name, params.description ]
+        , description: Just params.description
```

Comment:
This seems like an API change? will the frontend handle this correctly?

Reply:
The description and name aren't actually parsed from the metadata, they're retrieved from the database. However the image field is parsed from the metadata, and that format changed from an array to a string, so I updated the parsing for it. According to the cip25 standard the image field can be a string or an array, so I made the parsing handle both possibilities, and this also means the nfts minted before this update are still useable